app/controllers/citizens/consents_controller.rb

```diff
@@ -1,5 +1,31 @@
 module Citizens
   class ConsentsController < BaseController
-    def show; end
+    def show
+      @applicant = current_applicant
+    end
+
+    def create
+      @form = Applicants::OpenBankingConsentForm.new(edit_params)
+
+      @form.save
+
+      case @form.open_banking_consent
+      when 'true'
+        redirect_to applicant_true_layer_omniauth_authorize_path
+      when 'false'
+        render plain: 'Landing page: No Consent provided'
+        # redirect_to citizens_information_path
+      end
```

Comment:
`case` statements make sense if there are more than two options, and/or you want to do multiple `===` comparisons. `open_banking_consent` is a boolean! You should be able to do:
```
if @form.open_banking_consent?
  ...
else
  ...
end
```
Obviously you'll need to ensure `open_banking_consent?` return `false` for `"false"` Have a look at `ActiveModel::Type::Boolean.new.cast`

Reply:
removed the case statement and replaced with an if/else, it doesn't use the boolean operator though